canonical-web-and-design/maas-ui
# feat(legacy): remove legacy controllers list

## Done

- Remove the legacy controllers list.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the full client.
- Check that the legacy pages load without errors (subnets and controller details).

## Fixes

Fixes: canonical-web-and-design/app-tribe#620.
